Gemfile.lock

```diff
@@ -1,44 +0,0 @@
-GEM
```

Comment:
@ppeble Thank you for review thir PR.
I have seen 'ruby 2.1.0' test failed and investigated about it  too. 
Bundler checked dependencies and test was failed.
Please look at https://travis-ci.org/holidays/definitions/jobs/206037743#L215-L217 .

I thought it's not necessary too, and removed it.